main/src/sphexa/propagator.hpp

```diff
@@ -73,6 +76,8 @@ class Propagator
     //! average number of neighbors per particle
     size_t ng0_;
```

Comment:
The average number of neighbors per particle is a very useless magnitude. Tracking the number of neighbors on each particle is done because if there is an issue on a working code with a specific scenario, it might be due to one (or few) particles that get super high or super low amounts of neighbors. This will be basically unseeable on the average count. Why do we keep this magnitude? Is it used somewhere?

Reply:
This is a misnomer on my part, this should be called "target number of neighbors per particles". The smoothing lengths
are adapted each step to aim at this number.